crates/wadm/src/server/storage.rs

```diff
@@ -116,15 +118,21 @@ impl ModelStorage {
                         Ok(None) => return None,
                         Err(e) => return Some(Err(e)),
                     };
+                    #[allow(deprecated)]
                     Some(Ok(ModelSummary {
                         name: model_name,
                         version: manifest.current_version().to_owned(),
                         description: manifest.get_current().description().map(|s| s.to_owned()),
                         deployed_version: manifest.get_deployed().map(|m| m.version().to_owned()),
-                        // TODO(thomastaylor312): Actually fetch the status info from the stored
-                        // manifest once we figure it out
-                        status: StatusType::default(),
-                        status_message: None,
+                        // Since we are just fetching the manifest from the store, we need to wait
```

Comment:
Should we define a different type for a list response? Adding more of these fields feels like it makes it even more misleading as we don't know what the current status is at all. These could all be deployed, but it returns "waiting." I think a client method can fetch additional status and possibly call something like `ModelSummary::from(list_item)`

Reply:
Yeah, probably. I think we assumed that we would eventually infer this information from the stored manifest, but we just don't.